documentation/modules/cruise-control/proc-generating-optimization-proposals.adoc

```diff
@@ -23,7 +23,7 @@ You can then analyze the summary information in the optimization proposal and de
 +
 [source,yaml,subs="attributes+"]
 ----
-apiVersion: {KafkaApiVersion}
+apiVersion: kafka.strimzi.io/v1alpha1
```

Comment:
Should we have an attribute for this to future-proof?

Reply:
I added a new `:KafkaApiVersionAlpha:` attribute.

Reply:
What is the value of having all these as variables? I can understand the value as long as we had all resources using the same API version and upgrading them was as easy as changing the variable. But we have now many CRD and they have their own version lifecycle. So having the variable has no value. You would still need to go through the docs and replace the variables manually.  So I think we should actually drop this from everywhere and have the API version directly in the examples (this is also where #3084 leads)

Reply:
Alright, I see your point. I've removed the new variable from this PR.